Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY --from=build /usr/src/app/node_modules node_modules
 COPY --from=build /usr/src/app/dist dist
 COPY package.json ./
 ENV PG_META_PORT=8080
-CMD ["npm", "run", "start"]
+CMD ["node", "dist/server/server.js"]
```

Comment:
What was wrong with `npm start` (`run` can be omitted here)?
It's one more place you've to remember to update `dist/server/server.js` in case it's moved 👀

Reply:
`npm run start` wasn't passing down signals from the parent process. I tried it initially with a barebone `docker-compose.yml`.

```yaml
version: '3'

services:
  pgmeta:
    build: .
```

Reply:
may be related to this issue https://github.com/npm/cli/issues/6684

Reply:
🫠